### PR TITLE
Story/stat slides

### PR DESCRIPTION
**Pull request by kjanderson2**
FINALLY got these changes pushed up. There are some weird things with some files so I'm gonna rebase but could get eyes on this whenever is convenient

### CODE REVIEW

File: Android/app/src/main/java/com/mehequanna/mmiw/StatViewPagerFragment.kt [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentStatePagerAdapter
+import kotlinx.android.synthetic.main.fragment_stat_viewpager.*
+
+private const val NUM_PAGES = 4
+
+class StatViewPagerFragment : Fragment() {
```
Comment (Mehequanna) [inline Android/app/src/main/java/com/mehequanna/mmiw/StatViewPagerFragment.kt:14]: Do we need this class anymore?

File: Android/app/src/main/res/drawable/gradient_view.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle" >
+    <gradient
+        android:startColor="#CC000000"
+        android:centerColor="#00000000"
+        android:endColor="#3B000000"
+        android:angle="90"/>
+    <corners
+        android:radius="0dp"/>
+
+
```
Comment (Mehequanna) [inline Android/app/src/main/res/drawable/gradient_view.xml:11]: ❌ Delete the lines?

File: Android/app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,15 +1,35 @@
...
\ No newline at end of file
+        android:text="@string/next_slide"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        android:layout_marginBottom="24dp"
+        android:textSize="24sp"
+        android:textColor="@android:color/white"
+        android:layout_marginEnd="24dp" />
+
+    <com.mehequanna.mmiw.widgets.StatIndicatorViewGroup
```
Comment (Mehequanna) [inline Android/app/src/main/res/layout/activity_main.xml:27]: 🔥 I need to figure out how to create one of these :)

File: Android/app/src/main/res/layout/stat_indicator_group.xml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        android:layout_height="18dp"
+        android:layout_marginStart="2dp"
+        android:layout_marginEnd="2dp"
+        android:importantForAccessibility="no"
+        android:src="@drawable/stat_indicator_white"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@id/indicator3"
+        app:layout_constraintTop_toTopOf="parent" />
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (Mehequanna) [inline Android/app/src/main/res/layout/stat_indicator_group.xml:56]: Needs new line at the end.

File: Android/app/src/main/java/com/mehequanna/mmiw/StatSlideFragment.kt [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        view.statText.text = text
+    }
+
+    companion object {
+        const val ARG_BACKGROUND_RES = "backgroundRes"
+        const val ARG_STAT_TEXT = "statTextString"
+        const val ARG_PAGE_INDEX = "pageIndex"
+    }
+
+}
```
Comment (Mehequanna) [inline Android/app/src/main/java/com/mehequanna/mmiw/StatSlideFragment.kt:56]: Needs new line at the end.

File: Android/app/src/main/java/com/mehequanna/mmiw/widgets/StatIndicatorViewGroup.kt [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+            3 -> {
+                indicator1.setImageResource(R.drawable.stat_indicator_white)
+                indicator2.setImageResource(R.drawable.stat_indicator_white)
+                indicator3.setImageResource(R.drawable.stat_indicator_white)
+                indicator4.setImageResource(R.drawable.stat_indicator_red)
+            }
+        }
+    }
+
+}
```
Comment (Mehequanna) [inline Android/app/src/main/java/com/mehequanna/mmiw/widgets/StatIndicatorViewGroup.kt:45]: Needs new line at the end.

File: Android/app/src/main/res/drawable/gradient_view.xml [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+</shape>
```
Comment (Mehequanna) [inline Android/app/src/main/res/drawable/gradient_view.xml:12]: Needs new line at the end.

File: Android/app/src/main/res/layout/fragment_stat_slide.xml [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+        android:layout_marginStart="16dp"
+        android:layout_marginEnd="16dp"
+        android:layout_marginBottom="64dp"
+        android:textSize="36sp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        tools:text="@string/stat_text_1" />
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (Mehequanna) [inline Android/app/src/main/res/layout/fragment_stat_slide.xml:48]: Needs new line at the end.

File: Android/app/src/main/res/layout/fragment_stat_viewpager.xml [hunk 9]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+    <androidx.viewpager.widget.ViewPager
+        android:id="@+id/pager"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (Mehequanna) [inline Android/app/src/main/res/layout/fragment_stat_viewpager.xml:9]: Needs new line at the end.

**Review by Mehequanna — APPROVED** [review]
This looks awesome. Thanks!

File: Android/app/src/main/java/com/mehequanna/mmiw/StatViewPagerFragment.kt (hunk 1 above)
Comment (kjanderson2) [reply Android/app/src/main/java/com/mehequanna/mmiw/StatViewPagerFragment.kt:14]: No, but in my next PR that I'm putting up soon I will so if it's okay, I'm going to leave it for now

**Pull request merged**